nautobot/dcim/templates/dcim/manufacturer.html

```diff
@@ -17,6 +17,12 @@
                         <a href="{% url 'dcim:device_list' %}?manufacturer={{ object.name }}">{{ device_table.rows|length }}</a>
                     </td>
                 </tr>
+                <tr>
+                    <td>Cloud Accounts</td>
+                    <td>
+                        <a href="{% url 'cloud:cloudaccount_list' %}?provider={{ object.name }}">{{ account_table.rows|length }}</a>
+                    </td>
+                </tr>
```

Comment:
I might suggest that we make this (and the table below) conditional display, i.e. not show them at all for Manufacturers that don't have any cloud-accounts defined. I figure most deployments will have a lot of non-cloud Manufacturers (Cisco, Arista, etc.) and it would be a source of some confusion for users.